scipy/stats/stats.py

```diff
@@ -3192,6 +3198,21 @@ def f_oneway(*args):
     alldata = np.concatenate(args)
     bign = len(alldata)
 
+    # Check if each the values within each group are constant
```

Comment:
suggest change to: `# Check if the values within each group are constant`

Reply:
Oh, I must've missed it when changing the sentense. Sure!